akka-docs/src/main/paradox/cluster-usage.md

```diff
@@ -12,6 +12,47 @@ To use Akka Cluster, you must add the following dependency in your project:
   version="$akka.version$"
 }
 
+## When and where to use Akka Cluster
+
+An architectural choice you have to make is if you are going to use a microservices architecture or
+a traditional distributed application, also known as a distributed monolith.
+
+Microservices has many attractive properties, such as the independent nature of microservices allows for
+multiple smaller and more focused teams that can deliver new functionality more frequently and can
+respond quicker to business opportunities. Reactive Microservices should be isolated, autonomous, and have
+a single responsibility as identified by Jonas Bonér in the book
+[Reactive Microservices Architecture: Design Principles for Distributed Systems](https://info.lightbend.com/COLL-20XX-Reactive-Microservices-Architecture-RES-LP.html).
```

Comment:
This reference should probably be changed to Ruth's high level guide when it's ready.

Reply:
I would leave both in, positioning the new web pages as an introduction and Jonas' book as the detailed info.

Reply:
ok, I think I got the link from Lagom documentation ;-)

Reply:
Yeah, one of many things we need to update but haven't gotten to yet. Let's not propagate the error! 😉